### PR TITLE
Adding blacklist and whitelist

### PR DESCRIPTION
**Pull request by wuz**
There has been a lot of discussion around the usage of the words `blacklist` and `whitelist`. You can see more [here](https://github.com/graphite-project/graphite-web/issues/1569) and [here](https://github.com/kickstarter/rack-attack/pull/181). Using words like `blocklist` and `safelist` are more inclusive and more descriptive.

### CODE REVIEW

**Comment by wooorm** [comment]
Hey nice! Looks like this is an older branch, could you fix the conflicts first? 👍

**Comment by wuz** [comment]
Whoops! Not sure how I managed that. On it!

**Comment by wuz** [comment]
Fixed!

File: script/race.yml [hunk 1]
```diff
@@ -123,3 +123,19 @@
     - gypsy
     - Gipsy
     - gyp
+- type: simple
+  note: Replace racially-charged language with more accurate and inclusive words
+  considerate:
+    - blocklist
```
Comment (wooorm) [inline script/race.yml]: Nice! It does seem that the first suggestions aren’t antonyms, whereas 2 and 3 are. Why not blacklist/passlist or wronglist/alrightlist?

Comment (wuz) [reply script/race.yml]: Good question. I could definitely make that a thing. My thinking was that blacklist->blocklist and whitelist->alrightlist are sort of homonyms. I think what you recommended make sense though!

**Comment by wooorm** [comment]
I think this is nice, WDYT>

**Comment by wuz** [comment]
I like it!

**Pull request merged**